src/common/ceph_releases.cc

```diff
@@ -38,7 +38,7 @@ bool can_upgrade_from(ceph_release_t from_release,
   if (cutoff < to_release) {
     err << "recorded " << from_release_name << " "
         << to_integer<int>(from_release) << " (" << from_release << ") "
-        << "is >2 releases older than installed "
+        << "is 2 or more releases older than installed "
```

Comment:
this is wrong.

Reply:
Sigh... looks like this should read "is more than two releases older than installed"